coverage.txt

```diff
@@ -1,25 +1,27 @@
 Name                                         Stmts   Miss  Cover
 ----------------------------------------------------------------
 testing/__init__.py                              0      0   100%
-testing/test_all.py                              9      0   100%
+testing/test_all.py                             10      0   100%
 testing/yuca/__init__.py                         0      0   100%
 testing/yuca/ca/test_cca.py                    111     22    80%
 testing/yuca/ca/test_nca.py                     90     22    76%
 testing/yuca/ca/test_reaction_diffusion.py      45      1    98%
 testing/yuca/test_activations.py                34      0   100%
+testing/yuca/test_clone.py                      24      1    96%
 testing/yuca/test_configs.py                    38      4    89%
 testing/yuca/test_kernels.py                    36      1    97%
 testing/yuca/test_utils.py                     188      0   100%
 yuca/__init__.py                                 0      0   100%
-yuca/activations.py                            197     60    70%
+yuca/activations.py                            197     59    70%
 yuca/ca/__init__.py                              0      0   100%
-yuca/ca/common.py                              127     20    84%
-yuca/ca/continuous.py                          352     40    89%
+yuca/ca/common.py                              130     20    85%
+yuca/ca/continuous.py                          353     35    90%
 yuca/ca/neural.py                               74      0   100%
 yuca/ca/reaction_diffusion.py                   88      5    94%
+yuca/clone.py                                   81     11    86%
 yuca/configs.py                                 89     21    76%
 yuca/kernels.py                                 98     11    89%
 yuca/patterns.py                                59     43    27%
 yuca/utils.py                                  127      7    94%
 ----------------------------------------------------------------
-TOTAL                                         1762    257    85%
+TOTAL                                         1872    263    86%
```

Comment:
✓requirement for test_commit coverage for PRs
✓meets line coverage threshold target >= 85%